### ISSUE TITLE
Disabled-on-external-mouse send-event disables touchpad completely

### ISSUE THREAD

**Issue by pollaloca** [pop-os/pop#801]
**Distribution (run `cat /etc/os-release`):**
 DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=19.10
DISTRIB_CODENAME=eoan
DISTRIB_DESCRIPTION="Pop!_OS 19.10"
NAME="Pop!_OS"
VERSION="19.10"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.10"
VERSION_ID="19.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=eoan
UBUNTU_CODENAME=eoan
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
Setting touchpad's **send-event** with Dconf Editor to **'Disabled-on-external-mouse'** disables the touchpad completely, even when mouse is disconnected.

**Steps to reproduce (if you know):**

**Expected behavior:**
Touchpad should be disabled when external mouse is connected and should be enabled automatically when disconnecting the mouse.

**Other Notes:**
Lenovo Y540

**Comment by brs17** [comment]
I tried to recreate this issue on 3 different computers with 2 different mice and the setting behaved just as expected.

Have you tried rebooting after changing the setting?

**Comment by pollaloca** [comment]
Yes. I have rebooted the laptop several times since I found the problem. It happens with both bluetooth and USB mice.

**Comment by juankarlos999** [comment]
Touchpad  buttons options are not enabled within the browser. my operating system is pop_os 19.10

**Comment by brs17** [comment]
> Yes. I have rebooted the laptop several times since I found the problem. It happens with both bluetooth and USB mice. And if I go to settings and activate the touchpad (while mouse connected), it doesn't work.

I notice that if I activate the touchpad via GNOME Settings when the mouse is connected, the dconf setting changes from `disabled-on-external-mouse` to `enabled`.

This all seems to be working as I would expect on the machines/mice I have tested on. 

Perhaps you have some other dconf settings changed that are causing additional issues? Perhaps you could also try this on a fresh install/live image/recovery partition to see if that may in fact be the case.

**Comment by pollaloca** [comment]
I've just tried with a live USB and it appears to be working correctly. Not sure what can be interfering with this.

I've also notice that when booting up, in the login manager the touchpad works even if the mouse is connected

**Comment by brs17** [comment]
Potentially resetting dconf/gsettings may do the trick for you (`dconf reset -f /`). A word of caution however, it will revert this https://github.com/pop-os/default-settings/blob/master/usr/bin/pop-app-folders which you can just run again and should be back to defaults.

All of the Pop!_OS defaults are set here: https://github.com/pop-os/default-settings and https://github.com/pop-os/session

**Comment by pollaloca** [comment]
So, I've decided to follow the nuke&pave path but I still have the same problem.

This are the steps that I've followed:
 1.- Clean install of Pop_OS 19.10
 2.- sudo apt update 
 3.- sudo apt upgrade
 3.- reboot
 4.- sudo apt install dconf-editor
 5.- Setted Touchpad's 'send-event' to 'disable-on-external-mouse'
 6.- reboot
 7.- Disconneted external mouse
 8.- Touchpad not working

Also, for some reason, Lenovo logo appears after loging in:
1.- Login manager
2.- Lenovo logo screen appears
3.- black screen
4.- Lenovo logo screen appears again
5.- desktop appears.

**Comment by pollaloca** [comment]
Now I've done `dconf reset -f /` but the problem persists.

**Comment by pollaloca** [comment]
After tinkering a little bit, it appears that the problem comes from the bluetooth:
1.- New fresh clean install
2.- apt update, upgrade,...
3.- Reboot
4.- Install dconf-editor
5.- Set Touchpad's 'send-event' to 'disable-on-external-mouse'
6.- Plug USB mouse: mouse works / touchpad is disabled
7.- Unplug mouse: mouse doesn't work / touchpad works
8.- Enable bluetooth (from settings)
9.- Pair with bluetooth mouse: mouse works / touchpad is disabled
10.- Turn off  bluetooth mouse: mouse doesn't work / touchpad is disabled
11.- Reboot
10.- Plug USB mouse: mouse works / touchpad is disabled
11.- Unplug mouse: mouse doesn't work / touchpad works
